src/Feliz.Plotly/Feliz.Plotly.fsproj

```diff
@@ -5,6 +5,18 @@
     <PackageId>Feliz.Plotly.V2</PackageId>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Version>2.28.0</Version>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Authors>Kurt Mueller</Authors>
+    <PackageTags>fsharp fable react visualization graph chart</PackageTags>
+    <PackageDescription>Fable bindings written in the Feliz-style for plotly.js</PackageDescription>
+    <!-- you can fill in this property with whatever release notes you want users to see on
+    NuGet.Org -->
+    <PackageReleaseNotes></PackageReleaseNotes>
```

Comment:
The old package had a space for release notes, so I added this empty placeholder so you'd see there was a place for adding it.